projects/commudle-admin/src/app/feature-modules/user-follow/user-follow.component.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, EventEmitter, Input, OnDestroy, OnInit, Output, TemplateRef} from '@angular/core';
+import {IUser} from 'projects/shared-models/user.model';
+import {AppUsersService} from 'projects/commudle-admin/src/app/services/app-users.service';
+import {Subscription} from 'rxjs';
+import {ICurrentUser} from 'projects/shared-models/current_user.model';
+import {NbDialogRef, NbDialogService} from '@nebular/theme';
+
+@Component({
+  selector: 'app-user-follow',
+  templateUrl: './user-follow.component.html',
+  styleUrls: ['./user-follow.component.scss']
+})
+export class UserFollowComponent implements OnInit, OnDestroy {
+
+  @Input() user: IUser;
+  @Input() currentUser: ICurrentUser;
+
+  @Output() userFollowed: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Why's this needed? Since this is an independent component. Just askign

Reply:
Changed!